secantTests/ImportWalletTests/ImportWalletTests.swift

```diff
@@ -18,7 +18,7 @@ class ImportWalletTests: XCTestCase {
         let store = TestStore(
             initialState: .initial
         ) {
-            ImportWalletReducer(saplingActivationHeight: 0)
```

Comment:
I don't know if these tests were depending on this value of `saplingActivationHeight`; presumably the test dependency injection will now be normalizing these to the actual testnet Sapling activation height.

Reply:
`saplingActivationHeight` is newly read directly inside ImportWalletReducer from a dedicated TCA dependency